### DISCUSSION TITLE
Add new chapter

### DISCUSSION THREAD

**Discussion by rmfmarinho** [joaomlourenco/novathesis#443] [Q&A]
I need to add chapter 4 for conclusion, and i created it, i just dont know where to "call" it so that it shows on the main template. 

How do I do it?

Thanks for the help!

**Comment by joaomlourenco** [comment]
Please read the instructions in Chapter 1 of the original document.  Also please read the instructions in the wiki. Then, maybe, spend some of your time studying and understanding the files in the `Config` folder.  And I'm sure you'll get there.
